projects/igniteui-angular/src/lib/services/excel/excel-files.ts

```diff
@@ -81,7 +81,15 @@ export class WorksheetFile implements IExcelFile {
             sheetData.push('</row>');
 
             for (let i = 1; i < worksheetData.rowCount; i++) {
-                sheetData.push(`<row r="${(i + 1)}"${rowHeight}>`);
+                const rowData = worksheetData.data[i - 1].originalRowData;
+
+                const sCollapsed = (!rowData.expanded) ? '' : (rowData.expanded === true) ? '' : ` collapsed="1"`;
+                const sHidden = (rowData.parent && this.hasCollapsedParent(rowData)) ? ` hidden="1"` : '';
+                const rowOutlineLevel = rowData.level ? rowData.level : 0;
+                const sOutlineLevel = rowOutlineLevel > 0 ? ` outlineLevel="${rowOutlineLevel}"` : '';
+                maxOutlineLevel = maxOutlineLevel < rowOutlineLevel ? rowOutlineLevel : maxOutlineLevel;
```

Comment:
We should not depend on just verifying if there are properties like level, expanded, data, etc. since if you export a regular grid with a data containing properties with these names, the exported Excel file will be broken.